docs/_data/toc.yml

```diff
@@ -67,18 +67,32 @@
       url: concepts/architecture
     - title: Adapters
       url: concepts/architecture/adapters
+    - title: Broker
```

Comment:
"under Architecture" meaning indented under Architecture

Reply:
apologies, Done!
added rest of them under architecture except catalog subheading since its location is concepts/catalog